### PR TITLE
Reset password process v2

### PR DESCRIPTION
**Pull request by motu55**

### CODE REVIEW

**Comment by nixnuex** [comment]
Open questions: which mail function should be used?

**Comment by nixnuex** [comment]
Closes #174

File: juntagrico/config.py [hunk 1]
```diff
@@ -117,7 +117,6 @@ class Config:
             'welcome': 'mails/member/member_welcome.txt',
             'co_welcome': 'mails/member/co_member_welcome.txt',
             'co_added': 'mails/member/co_member_added.txt',
-            'password': 'mails/member/password_reset.txt',
```
Comment (motu55) [inline juntagrico/config.py:120]: why not leave it here and use the config value in the urls.py?

File: juntagrico/templates/registration/password_reset_complete_cust.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% extends "no_menu_base.html" %}
+{% load i18n %}
+{% load juntagrico.config %}
+{% block allcontent %}
+    <div class="offset-md-3 col-md-6">
+        <div class="alert alert-success mb-3">
+            <p>{% translate "Your password has been set.  You may go ahead and log in now." %}</p>
```
Comment (motu55) [inline juntagrico/templates/registration/password_reset_complete_cust.html:7]: leaving this in english will probably lead to some errors when we create the gettext files
i would insert the german translations

File: juntagrico/urls.py [hunk 3]
```diff
@@ -30,13 +31,19 @@
     # logout/
     path('logout/', juntagrico.logout_view, name='logout'),
 
+    # /accounts (password reset)
+    path('accounts/password_reset/', auth_views.PasswordResetView.as_view(template_name='registration/password_reset_form_cust.html',
+                                                                          email_template_name='mails/member/password_reset_email.txt'), name='password_reset'),
```
Comment (motu55) [inline juntagrico/urls.py:36]: why not use the config here?

**Pull request merged**